OpenRA.Mods.Common/Effects/PowerdownIndicator.cs

```diff
@@ -33,7 +33,7 @@ public PowerdownIndicator(Actor a)
 
 		void IEffect.Tick(World world)
 		{
-			if (!a.IsInWorld || a.IsDead || !canPowerDown.Disabled)
+			if (!a.IsInWorld || a.IsDead || !canPowerDown.PoweredDown)
```

Comment:
This whole class can be removed, and replaced by a conditional `WithDecoration` (see e.g. D2K carryall).